com.unity.sg2/Tests/GraphUI/BaseGraphWindowTest.cs

```diff
@@ -24,8 +24,8 @@ public class BaseGraphWindowTest
         protected virtual bool hideOverlayWindows => true;
 
         // Need to match the values specified by the BlackboardOverlay and ModelInspectorOverlay in GTFO
-        const string k_BlackboardOverlayId = "gtf-blackboard";
-        const string k_InspectorOverlayId = "gtf-inspector";
+        const string k_BlackboardOverlayId = SGBlackboardOverlay.k_OverlayID;
```

Comment:
Oh good catch with this, I need to get into the habit of running the UI tests and posting the results like you've been doing to ensure no regressions